training-manager.lic

```diff
@@ -118,7 +109,7 @@ class TrainingManager
   def hunting_combo
     wait_for_script_to_complete('hunting-buddy')
     wait_for_script_to_complete('safe-room')
-    wait_for_script_to_complete('crossing-repair') if check_repair
+    wait_for_script_to_complete('crossing-repair', ['force']) if check_repair
```

Comment:
Since `crossing-repair` will now have logic to run no more often than every N seconds, if `training-manager` decides the script should run because X hunts have been performed then let it force the repair script to run.